tools/devops/automation/templates/build/stage.yml

```diff
@@ -143,12 +143,13 @@ jobs:
     parameters:
       devicePrefix: sim
       runTests: ${{ parameters.runTests }}
+      apiGeneratorDiffBuilt: $[ dependencies.build.outputs['apiGeneratorDiff.API_GENERATOR_BUILT'] ]
```

Comment:
YAML is a mess, but at this stage.. dependencies is not know if we use ${{ }} os we have to use the runtime $[ ] and $[ ] is never a boolean is always a string, so we cannot do a contains() because the expression is a string.. until runtime!!!

WELL DONE 

/me slow clap